### PR TITLE
Biblio ui

### PR DESCRIPTION
**Pull request by amitaibu**

### CODE REVIEW

File: biblio_ui/biblio_ui.module [hunk 1]
```diff
@@ -0,0 +1,639 @@
...
+  $account = (!$account) ? user_load($user->uid) : user_load($account->uid);
+
+  $query = new entityFieldQuery();
+  $number = $query
+    ->entityCondition('entity_type', 'biblio')
+    ->propertyCondition('type', $bundle)
+    ->count()
+    ->execute();
+
+  if ($number > 0 || !user_access('manage biblio structure')) {
```
Comment (amitaibu) [inline biblio_ui/biblio_ui.module]: single line.

File: biblio_ui/biblio_ui.module [hunk 2]
```diff
@@ -0,0 +1,639 @@
...
+  ));
+}
+
+/**
+ * Implements hook_entity_info_alter().
+ */
+function biblio_ui_entity_info_alter(&$info) {
+  $biblio_types = biblio_types();
+
+  if (empty($biblio_types)) {
```
Comment (amitaibu) [inline biblio_ui/biblio_ui.module]: Move inside if

File: biblio_ui/biblio_ui.module [hunk 3]
```diff
@@ -0,0 +1,639 @@
...
+        'access arguments' => array('manage biblio structure'),
+      ),
+    );
+  }
+}
+
+/**
+ * Manage the biblio entries in you site.
+ */
+function biblio_ui_manage() {
```
Comment (amitaibu) [inline biblio_ui/biblio_ui.module]: Remove, we will use Views.

**Comment by RoySegall** [comment]
Merged.

**Pull request closed without merging**